apps/form-system/web/infra/form-system-web.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  CodeOwners,
+  ref,
+  service,
+  ServiceBuilder,
+} from '../../../../infra/src/dsl/dsl'
+
+const serviceName = 'form-system-web'
+export const serviceSetup = (): ServiceBuilder<typeof serviceName> =>
+  service(serviceName)
+    .codeOwner(CodeOwners.Advania)
+    .namespace(serviceName)
+    .liveness('/liveness')
+    .readiness('/readiness')
+    .replicaCount({
+      default: 2,
+      max: 30,
+      min: 2,
+      scalingMagicNumber: 8,
+    })
```

Comment:
Do you know the load of this application, or is this just a guess? If guessing, you probably don't need that many replicas, and the scaling number is deprecated.
```suggestion
    .replicaCount({
      default: 2,
      max: 4,
      min: 2,
    })
```